gradle/maven-push.gradle

```diff
@@ -16,11 +16,11 @@ def getSnapshotRepositoryUrl() {
 }
 
 def getRepositoryUsername() {
-    return hasProperty("NEXUS_USERNAME") ? NEXUS_USERNAME : ""
+    return hasProperty("SONATYPE_USERNAME") ? NEXUS_USERNAME : ""
```

Comment:
@brianPlummer Ah, this needs to be `return hasProperty("SONATYPE_USERNAME") ? SONATYPE_USERNAME : ""` and `return hasProperty("SONATYPE_PASSWORD") ? SONATYPE_PASSWORD : ""`.